cmd/cloudflared/main.go

```diff
@@ -482,10 +482,11 @@ func startServer(c *cli.Context, shutdownC, graceShutdownC chan struct{}) error
 		return err
 	}
 
+	metricsLabels := map[string]string{"application": "cloudflared"}
```

Comment:
What's the reason to add this label when the metrics names are prefix with argotunnel?

Reply:
It's there mostly as an example of how to add additional labels, but it could distinguish cloudflared from other applications (the ingress controller) that are using the library.

Reply:
Is namespace not enough to distinguish cloudflared from other applications?